apps/condo/domains/notification/gql.js

```diff
@@ -23,10 +23,27 @@ const RESEND_MESSAGE = gql`
         result: resendMessage(data: $data) { status id }
     }
 `
+
+const getPushTokenFields = (extra = 'id') => `{ deviceId token serviceType ${extra} }`
```

Comment:
it's too complex to test it by `/admin/api` you need to execute the function in your brain %)

Reply:
fixed